.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy to Cocoapods
+
+on:
+  release:
```

Comment:
if you are doing this here, then you'll need to remove the _push to bintray_ step in the `release.yml` workflow and update the name property to Create Github Release